src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.scss

```diff
@@ -47,14 +47,16 @@
     float: left;
   }
 
-  ::ng-deep .table-filters {
-    border-left: 1px solid bd.$datatable-divider-color;
-    float: right;
-    padding-left: 8px;
-  }
+  ::ng-deep cd-table {
+    .table-filters {
+      border-left: 1px solid bd.$datatable-divider-color;
+      float: right;
+      padding-left: 8px;
+    }
 
-  ::ng-deep .table-filters label {
-    margin-right: 4px;
+    .table-filters label {
+      margin-right: 4px;
+    }
```

Comment:
`table-filters` can be removed since we no longer use this.
Please remove it from the scss and html.